wordpress-mobile/WordPress-iOS
# Issue/aztec insert videos

Fixes #7289 

This PR adds support to be able to insert video on aztec using the insert media button on the toolbar.

To test:
 - Start a new post
 - Tap on add media
 - Select a video or two
 - See if the upload happens correctly
 - Tap on the video and see if it plays
 - Switch to HTML and see if the code is correct:
   - [wpvideo] for WP.com
   - [video] for self hosted
 - Switch to visual again and see if they display correctly
 - Test on WP.com site and self-hosted.

Needs review: @jleandroperez
